src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+* and values are the values of related CSS
+ properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splitString = sourceString.split(';').map(elem => elem.split(':'));
+
+  const delEpmty = splitString.filter(elem => elem.length !== 1);
+
+  const delSpaces = delEpmty.map(elem => elem.map(item => item.trim()));
+
+  delSpaces.map(elem => {
+    result[elem[0]] = elem[1];
```

Comment:
u can trim elements here

Reply:
I don't understand how to implement it and why it can't be like I wrote

Reply:
It is an option actually, I approved your solution, but it would make complexity of your solution easier